en/_posts/2016-06-23-linux-and-skylake.md

```diff
@@ -0,0 +1,160 @@
+
+
+## Method 2: use a precompiled bzImage proposed by ovh
+
+You can try out our kernels and see if they suit your needs.
+
+[Go to our bzImages repository](ftp://ftp.ovh.net/made-in-ovh/bzImage/)
+
+
+For example you may be interested in
+
+[bzImage-4.5.2-mod-std-ipv6-64](ftp://ftp.ovh.net/made-in-ovh/bzImage/4.5.2/bzImage-4.5.2-mod-std-ipv6-64)
+
+Or (similar to the low latency one showed in method 1, except it has got the drawback of not allowing dynamic module load):
+
+[bzImage-4.5.2-xxxx-std-ipv6-64-hz1000](ftp://ftp.ovh.net/made-in-ovh/bzImage/4.5.2/bzImage-4.5.2-xxxx-std-ipv6-64-hz1000)
+
+
+Please follow the already [existing guide](http://help.ovh.com/KernelInstall) for this, section 'Procedure to up date the kernel on your hard disk'
+
+ 
+## Method 3: custom compilation and deployment
+
+
+The most tedious thing is to build the initial kernel configuration when you start from scratch. 
+So we attach here some configurations which you can use as a starting point
+
+* [generic](/kb/attachments/2016-06-23-linux-and-skylake/config-4.4.0-generic)
+
+* [lowlatency](/kb/attachments/2016-06-23-linux-and-skylake/config-4.4.0-lowlatency)
+
+Alternatively you can also take the one from your running kernel: 
+
+Depending on its configuration (CONFIG\_PROC\_FS=y, CONFIG\_IKCONFIG=y, CONFIG\_IKCONFIG\_PROC=y):
+
+```
+# zcat /proc/config.gz > /tmp/mystartingconfig
+```
+
+Or if not enabled:
+
+```
+# find /boot -type f -name "config-$(uname -r)"
+```
+
+so now you have your config to start from
+
+Go to kernel.org and grab latest sources, for example `export $KERNEL_VERSION=4.6.2`
+
+
+```
+# apt-get install -y gcc make libncurses5-dev libssl-dev bc
+# wget https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-${KERNEL_VERSION}.tar.xz
+# tar xf linux-${KERNEL_VERSION}.tar.xz
+# cd linux-${KERNEL_VERSION}
+# cp /tmp/mystartingconfig .config
+# make olddefconfig
+# export THREADS=$(lscpu | grep -E '^CPU\(s\):' | awk '{print $2}')
+# make -j $THREADS bzImage modules
+# make modules_install
+# make install
+```
+
+`make install` should take care of building your initramfs and updating you bootloader config files
+
+## Enjoy
+
+```
+# lscpu
+Architecture:          x86_64
+CPU op-mode(s):        32-bit, 64-bit
+Byte Order:            Little Endian
+CPU(s):                8
+On-line CPU(s) list:   0-7
+Thread(s) per core:    2
+Core(s) per socket:    4
+Socket(s):             1
+NUMA node(s):          1
+Vendor ID:             GenuineIntel
+CPU family:            6
+Model:                 94
+Model name:            Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
+Stepping:              3
+CPU MHz:               800.000
+CPU max MHz:           4700.0000
+CPU min MHz:           800.0000
+BogoMIPS:              8016.72
+Virtualization:        VT-x
+L1d cache:             32K
+L1i cache:             32K
+L2 cache:              256K
+L3 cache:              8192K
+```
+
+## Contacts:
+
+* mailing list: **hardware.labs@ml.ovh.net** (you need to subscribe to post there)
```

Comment:
I tend to link to the subscribe variant directly to avoid searching how to subscribe hardware.labs-subscribe@ml.ovh.net